### ISSUE TITLE
Chocolatey package

### ISSUE THREAD

**Issue by jzabroski** [Phaiax/PxKeystrokesForScreencasts#45]
Please create a http://chocolatey.org package for this

**Comment by Phaiax** [comment]
I can do this. But I am currently porting the app to WPF to support multi monitor / mutli dpi setups. There will be a chocolately package for the new release.

**Comment by teknowledgist** [comment]
I'd be happy to build a Chocolatey package for the current version if you would like.  I try to "do right" by [my packages](https://community.chocolatey.org/profiles/teknowledgist).  By Chocolatey.org policy and as the author, you would always retain the right to take over (or share) maintainership whenever you are ready.

**Comment by Phaiax** [comment]
Fine for me :) 

Am 12. Oktober 2021 19:09:50 MESZ schrieb teknowledgist ***@***.***>:

**Comment by Phaiax** [comment]
👍

Am 14. Oktober 2021 19:41:34 MESZ schrieb teknowledgist ***@***.***>:
>[Here ya' go](https://community.chocolatey.org/packages/key-n-stroke)  Let me know (fairly soon) if you would like any of the information changed.  (It can be changed any time, but the package version has to be incremented once a submission has been approved.)
>
>BTW, if you can start using the "Releases" option here, I can easily make it an "automatic" package that will update when it sees that a new version is available.
>
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/Phaiax/Key-n-Stroke/issues/45#issuecomment-943577953
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by rollingmoai** [comment]
> BTW, if you can start using the "Releases" option here, I can easily make it an "automatic" package that will update when it sees that a new version is available.

@Phaiax Have you done this already? It seems like you still but the binaries on the source code itself.

**Comment by teknowledgist** [comment]
Just FYI, but I totally missed that there was a new version for 5.5 months until the recent post reminded me to take a look.  I've submitted a new Chocolatey package to the public repository for moderation, but if the binaries (or just standardized announcements) start appearing in "Releases", I can get a new package built/submitted within days of release (hours if I ever get around to running a headless updater).

**Comment by rollingmoai** [comment]
Yep, binaries inside source code are a bad idea in the first place.

**Comment by jzabroski** [comment]
Ideally it is generated via a GitHub Action as part of generating a Release.

**Comment by jzabroski** [comment]
https://github.com/marketplace/actions/chocolatey-action
